public/assets/components.css.map

```diff
@@ -1 +1 @@
-{"mappings":"AAKE;;;;;;;AAOD;;;;AAQC;;;;;;;AASA;;;;;AAOA;;;;;;AAMA;;;;;AAIA;;;;;AAUA;;;;;;AAKA;;;;AAMA;;;;;AAQA;;;;AAIA;;;;;;;AAMD;;;;AAEA;;;;AAMC;;;;;;AAQA;;;;;;;;;;AAcA;;;;AAMA;;;;;;AASA;;;;AAIA;;;;AAIA;;;;AAIA;;;;AAMA;;;;;AAKA;;;;AAKA;;;;;AAKA;;;;AAIA;;;;AAcD;;;;;AAGA;;;;AAEA;;;;;;AAQC;;;;AAKA;;;;;AAMA;;;;AAKA;;;;AAMA;;;;;AAYA;;;;;AAID;;;;;;;AAKA;;;;;;;AAIA;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;AA6CA;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;AA6CA;;;;AAIA;;;;AAGA;EACC;;;;EAKA;;;;;AAID;;;;;AAGA;;;;;AAGA;;;;;AAGA;;;;;;;;;;;;;;;AAaA;;;;AAEA;;;;AAEA;;;;AAEA;;;;AAEA;;;;AAEA;;;;;;AAIA;;;;;;;;;;;;;AAWA;;;;;AAGA;;;;;AAGA;;;;;;;;;AAOA;;;;;;;;;AAOA;;;;;AAGA","sources":["styles/components.css"],"sourcesContent":["@import \"./components/elements.css\";\n@import \"./components/select.css\";\n@import \"./components/checkbox.css\";\n@import \"./components/rating.css\";\n@import \"./components/accordion.css\";\n@import \"./components/toggle.css\";\n@import \"./components/tabs.css\";\n"],"names":[],"version":3,"file":"components.css.map","sourceRoot":"../../src/"}
```

Comment:
could you please `git rm` these map files? `npm run build` shouldn't generate the map files, if it does please let me know and i'll take a look & fix it

Reply:
hm, it was there: https://github.com/Platform-OS/pos-module-components/blob/master/.gitignore#L9
but i think we already had somthing in the repo by the time we added it to gitignore, or maybe the gitignore pattern is not right?